src/presentation/http/router/noteSettings.ts

```diff
@@ -57,7 +57,7 @@ const NoteSettingsRouter: FastifyPluginCallback<NoteSettingsRouterOptions> = (fa
   fastify.get<{
     Params: GetNoteSettingsByNodeIdOptions,
```

Comment:
this interface still has `id`. I'd suggest to get rid of it and move param type in-place.